sdk/storage/storage-queue/src/generated/src/models/index.ts

```diff
@@ -671,11 +671,6 @@ export interface MessageIdDeleteMethodOptionalParams extends coreHttp.RequestOpt
  * Defines headers for SetProperties operation.
  */
 export interface ServiceSetPropertiesHeaders {
-  /**
-   * If a client request id header is sent in the request, this header will be present in the
-   * response with the same value.
-   */
-  clientRequestId?: string;
```

Comment:
1

Reply:
I'm not sure where these went as we generally strip them out in .NET anyway.  We can add them all back via a transform.

Reply:
Added